test_conformance/printf/test_printf.cpp

```diff
@@ -214,15 +223,148 @@ int waitForEvent(cl_event* event)
 // Static helper functions definition
 //-----------------------------------------
 
+float get_random_float(float low, float high, MTdata d)
+{
+    float t = (float)((double)genrand_int32(d) / (double)0xFFFFFFFF);
+    return (1.0f - t) * low + t * high;
+}
+
+//-----------------------------------------
+// makeMixedFormatPrintfProgram
+//-----------------------------------------
+cl_program makeMixedFormatPrintfProgram(cl_kernel* kernel_ptr,
+                                        const cl_context context,
+                                        const cl_device_id device,
+                                        const unsigned int testId,
+                                        const std::string& testname)
+{
+    auto gen_char = [&]() {
+        static const char dict[] = {
+            " !#$&()*+,-./"
```

Comment:
`\t` added, `\n` is kind of a hassle

Reply:
I would suggest writing the tab character as `\t` literally (it'll still be one actual character in the string), as otherwise it may gets lost at some point when someone's editor converts tabs to spaces.

Fine to leave `\n` out here, I can imagine it could lead to hassle and we already have it covered elsewhere anyway.

Reply:
corrected